charts/tensorleap/templates/rabbitmq-cm.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
```

Comment:
do we really needs this?

Reply:
It's more for debugging, won't we need the management gui?

Reply:
This is the line that enables the `GUI`?
If so then it's fine :)

Reply:
Are you sure the file is in the right path?

Reply:
Yep, why?

Reply:
I don't see it the docs

Reply:
https://www.rabbitmq.com/management.html#usage-ui
https://www.rabbitmq.com/plugins.html#enabled-plugins-file

Reply:
nevermind, found it